ui-tests/src/test/java/org/openmrs/reference/page/HtmlFormsPage.java

```diff
@@ -0,0 +1,53 @@
+package org.openmrs.reference.page;
+
+import org.openmrs.Person;
+import org.openmrs.uitestframework.page.Page;
+import org.openqa.selenium.By;
+
+public class HtmlFormsPage extends Page {
+    private HomePage homePage;
+    private static final By NEW_HTML_FORM = By.cssSelector("a[href='htmlForm.form']");
+    private static final By NAME = By.name("form.name");
+//    private static  By DESCRIPTION = By.name("form.description");
+    private static By DESCRIPTION = By.cssSelector("#content textarea[name='form.description']");
+    private static  By SAVE_FORM = By.cssSelector("input[type=\"submit\"]");
+    private  static By VERSION = By.name("form.version");
+
+    public HtmlFormsPage(ManageHtmlFormsPage page) {
+        super(page);
+    }
+
+    public HtmlFormsPage(Page page) {
+        super(page);
+    }
+
+    public void setNAME(String name) throws InterruptedException {
+        findElement(NAME).clear();
+        findElement(NAME).sendKeys("newFormTest");
+        findElement(NAME).getAttribute("value");
+    }
+    public  void setDESCRIPTION(String description) {
+        findElement(DESCRIPTION).clear();
+        findElement(DESCRIPTION).sendKeys("description");
```

Comment:
you can use `setTextToFieldNoEnter` for sendKeys

Reply:
Actualy using findElement() seems to be more readable than setTextToFieldNoEnter

Reply:
Resolved thanks, cc @k-joseph  , any more comments on this pull request besides saving a form